lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,59 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    collection.count{ |element| (element.is_a? Float) || (element.is_a? Fixnum)}
```

Comment:
This works, but an easier way is use Numeric because it includes both Floats and Fixnums.